helpers/tf_cfg.py

```diff
@@ -179,6 +179,7 @@ def defaults(self):
                     "clickhouse_port": "8123",
                     "clickhouse_username": "default",
                     "clickhouse_password": "",
+                    "clickhouse_database": os.getenv("CLICKHOUSE_DATABASE", "default"),
```

Comment:
Please add `CLICKHOUSE_USERNAME` and `CLICKHOUSE_PASSWORD`. I would like to check these params on our CI.